content/en/docs/concepts/cluster-administration/flow-control.md

```diff
@@ -782,6 +782,100 @@ APF adds the following two headers to each HTTP response message.
+hold those seats for a shorter duration. List requests can occupy more than 1 
+seat depending on the number of objects fetched during the request. Restricting 
+the number of objects retrieved in a list request (by using pagination or 
+limiting the request to a particular namespace) will use less total seats over a
+shorter period. Furthermore, replacing list requests with watch requests will 
+require lower total concurrency shares as watch requests only occupy 1 seat 
+during its initial burst of notifications.
+
+### Priority and fairness settings {#good-practice-apf-settings}
+You can also modify the default FlowSchema and PriorityLevelConfiguration 
+objects or create new objects of these types to better accommodate your 
+workload.
+
+APF settings can be modified to:
+- Give more seats to high priority requests.
+- Isolate non-essential or expensive requests that would starve a concurrency
+level if it was shared with other flows.
+
+#### Give more seats to high priority requests
+1. You can create a new FlowSchema which references a PriorityLevelConfiguration 
+with a larger concurrency level. This new PriorityLevelConfiguration could be an 
+existing level or a new level with its own set of nominal concurrency shares. 
+Creating a new PriorityLevelConfiguration will reduce the number of seats 
+designated for existing levels. 
+2. You can also increase the NominalConcurrencyShares for the 
+PriorityLevelConfiguration which is serving your high priority requests. 
+Alternatively, you can increase the LendablePercent for competing priority 
+levels so that the given priority level has a higher pool of seats it can borrow.
+
+#### Isolate non-essential requests from starving other flows
+For request isolation, you can create a FlowSchema whose subject matches the 
+user making these requests or create a FlowSchema that matches what the request 
+is (corresponding to the resourceRules). Next, you can map this FlowSchema to a 
+PriorityLevelConfiguration with a low share of seats.
+
+For example, suppose list event requests from Pods running in the default namespace 
+are using 10 seats each and execute for 1 minute. To prevent these expensive 
+requests from impacting requests from other Pods using the existing service-accounts
+FlowSchema, you can apply the following FlowSchema to isolate these list calls 
+from other requests.
+
+Example FlowSchema object to isolate list event requests:
+```
+apiVersion: flowcontrol.apiserver.k8s.io/v1beta1
+kind: FlowSchema
+metadata:
+  name: list-events-default-service-accounts
+spec:
+  distinguisherMethod:
+    type: ByUser
+  matchingPrecedence: 8000
+  priorityLevelConfiguration:
+    name: catch-all
+  rules:
+  - resourceRules:
+    - apiGroups:
+      - '*'
+      namespaces:
+      - default
+      resources:
+      - events
+      verbs:
+      - list
+    subjects:
+    - kind: ServiceAccount
+      serviceAccount:
+        name: default
+        namespace: default
```

Comment:
Please use a `code` shortcode - see https://kubernetes.io/docs/contribute/style/hugo-shortcodes/#source-code-files.

Ideally, also change all the other examples in the same document (possibly by adding an extra commit to do only that).

Reply:
I added the list-events-default-service-account.yaml file and used the code shortcode. The only other FlowSchema or PriorityLevelConfiguration on this page is the health-for-strangers.yaml file which is already using this shortcode.